src/hotspot/cpu/x86/x86.ad

```diff
@@ -3879,6 +3880,18 @@ instruct roundD_reg(legRegD dst, legRegD src, immU8 rmode) %{
   ins_pipe(pipe_slow);
 %}
 
+instruct roundD_reg_avx(legRegD dst, legRegD src, immU8 rmode) %{
+  predicate(UseAVX > 0);
```

Comment:
can you push the predicate in instruction encoding block and fold this pattern with roundD_reg.